docs/src/examples/graf_files.md

```diff
@@ -1,9 +1,108 @@
-# Graf Files
+# Graf Files and Time Series
 
-## Writing a time series into a file
+## Time Series
+
+Some attributes in a Study represent a time series indexed by another attribute. Here we will be setting a time series for the attribute `HourDemand`, from `PSRDemandSegment`, which is indexed by `DataHourDemand`.
+
+First we create a Dict with `HourDemand` and `DataHourDemand` data.
+
+```@example rw_file
+series = Dict{String,Vector}(
+    "DataHourDemand" => ["1900-01-01", "1900-01-02","1900-01-03"],
+    "HourDemand" =>  [5.0, 7.0, 11.0]
+)
+```
```

Comment:
HourDemanda is a bad example because it should be hourly (and not daily as above).

Reply:
Changed to `EmissionCost` and `DateEmissionCost` from `PSRGasEmission`